Cargo.toml

```diff
@@ -56,54 +56,54 @@ lemmy_routes = { version = "=0.18.1-rc.10", path = "./crates/routes" }
 lemmy_db_views = { version = "=0.18.1-rc.10", path = "./crates/db_views" }
 lemmy_db_views_actor = { version = "=0.18.1-rc.10", path = "./crates/db_views_actor" }
 lemmy_db_views_moderator = { version = "=0.18.1-rc.10", path = "./crates/db_views_moderator" }
-activitypub_federation = { version = "0.4.4", default-features = false, features = ["actix-web"] }
+activitypub_federation = { version = "0.4.5", default-features = false, features = ["actix-web"] }
 diesel = "2.1.0"
 diesel_migrations = "2.1.0"
 diesel-async = "0.3.1"
-serde = { version = "1.0.164", features = ["derive"] }
-serde_with = "1.14.0"
+serde = { version = "1.0.167", features = ["derive"] }
+serde_with = "3.0.0"
 actix-web = { version = "4.3.1", default-features = false, features = ["macros", "rustls", "compress-brotli", "compress-gzip", "compress-zstd"] }
 tracing = "0.1.37"
-tracing-actix-web = { version = "0.6.2", default-features = false }
+tracing-actix-web = { version = "0.7.5", default-features = false }
 tracing-error = "0.2.0"
 tracing-log = "0.1.3"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 url = { version = "2.4.0", features = ["serde"] }
 url_serde = "0.2.0"
 reqwest = { version = "0.11.18", features = ["json", "blocking"] }
 reqwest-middleware = "0.2.2"
-reqwest-tracing = "0.4.4"
-clokwerk = "0.3.5"
+reqwest-tracing = "0.4.5"
+clokwerk = "0.4.0"
 doku = { version = "0.21.1", features = ["url-2"] }
-bcrypt = "0.13.0"
+bcrypt = "0.15.0"
 chrono = { version = "0.4.26", features = ["serde"], default-features = false }
-serde_json = { version = "1.0.96", features = ["preserve_order"] }
-base64 = "0.13.1"
-uuid = { version = "1.3.4", features = ["serde", "v4"] }
-async-trait = "0.1.68"
+serde_json = { version = "1.0.100", features = ["preserve_order"] }
+base64 = "0.21.2"
+uuid = { version = "1.4.0", features = ["serde", "v4"] }
+async-trait = "0.1.71"
 captcha = "0.0.9"
 anyhow = { version = "1.0.71", features = ["backtrace"] } # backtrace is on by default on nightly, but not stable rust
 diesel_ltree = "0.3.0"
-typed-builder = "0.10.0"
-serial_test = "0.9.0"
-tokio = { version = "1.28.2", features = ["full"] }
-sha2 = "0.10.6"
-regex = "1.8.4"
+typed-builder = "0.15.0"
+serial_test = "2.0.0"
+tokio = { version = "1.29.1", features = ["full"] }
+sha2 = "0.10.7"
+regex = "1.9.0"
 once_cell = "1.18.0"
 diesel-derive-newtype = "2.1.0"
 diesel-derive-enum = {version = "2.1.0", features = ["postgres"] }
-strum = "0.24.1"
-strum_macros = "0.24.3"
-itertools = "0.10.5"
+strum = "0.25.0"
+strum_macros = "0.25.1"
+itertools = "0.11.0"
 futures = "0.3.28"
 http = "0.2.9"
 percent-encoding = "2.3.0"
-rosetta-i18n = "0.1.2"
+rosetta-i18n = "0.1.3"
 rand = "0.8.5"
-opentelemetry = { version = "0.17.0", features = ["rt-tokio"] }
-tracing-opentelemetry = { version = "0.17.4" }
-ts-rs = { version = "6.2", features = ["serde-compat", "format", "chrono-impl"] }
```

Comment:
This format feature was causing build failures after updating deps. I dont think its really important.